libs/ledger-live-common/src/families/elrond/js-buildSubAccounts.ts

```diff
@@ -75,10 +75,12 @@ async function syncESDTTokenAccountOperations(
   );
   const operations = mergeOps(oldOperations, newOperations);
 
-  tokenAccount.operations = operations;
-  tokenAccount.operationsCount = operations.length;
+  if (operations === oldOperations) return tokenAccount;
 
-  return tokenAccount;
+  const copy = { ...tokenAccount };
```

Comment:
This only does a shallow copy, right? Can't that change the behavior?

Reply:
it should properly make the ref change when operations changes, so it behaves like all other coins 👍